pkg/config/config.go

```diff
@@ -55,6 +56,7 @@ type Config struct {
 	LogLevel                 string        `yaml:"logLevel"`
 	EnabledPlugin            []string      `yaml:"enabledPlugin"`
 	MetricsInterval          time.Duration `yaml:"metricsInterval"`
+	MetricsIntervalDuration  time.Duration `yaml:"metricsIntervalDuration"`
```

Comment:
or this one, whichever one shouldn't be referenced outside of the config

Reply:
Inside the go code `MetricsIntervalDuration` will not be referenced however in the yaml `metricsInterval` is deprecated, we should in future make the field like:
```
MetricsInterval    time.Duration    `yaml:"metricsIntervalDuration"`
```